index.md

```diff
@@ -1,277 +1,71 @@
 ---
 layout: lesson
 root: .
-lastupdated: Month Day, Year
-contributors: ["First Last", "First Last", "Pls Add Others"]
-maintainers: ["First Last", "First Last"]
-domain: Domain Name
-topic: Topic
-software:
-dataurl:
-status: Under Development
 ---
 
-Data Carpentry workshops are for any researcher who has data they want to analyze, and no prior computational experience is required. 
-This hands-on workshop teaches basic concepts, skills and tools for working more effectively with data.
-
-The focuses of this workshop will be working with genomics data, and data management & analysis for genomics research. We will cover
-metadata organization in spreadsheets, data organization, connecting to and using cloud computing, the command line for sequence quality
-control and bioinformatics workflows, and R for data analysis and visualization. We will not be teaching any particular bioinformatics 
-tools, but the foundational skills that will allow you to conduct any analysis and analyze the output of a genomics pipeline.
+Data Carpentry’s aim is to teach researchers basic concepts, skills, and tools for working
+with data so that they can get more done in less time, and with less pain. This workshop
+teaches data management and analysis for genomics research including: 
+best practices for organization of bioinformatics projects and data, use of command line 
+utilities, and connecting to and using cloud computing. Depending on which workshop
+track you tack, we will also cover either a) data analysis and visualization in the R 
+programming language or b) the use of command line tools to analyze sequence quality and 
```

Comment:
both versions do FASTQC, so I think it would be 

cover either a) data analysis and visualization in the R programming language or 2) automating bioinformatics workflows at the command line

Reply:
@tracykteal - My understanding is that only the "pipeline workflow" includes the wrangling lesson, which is where all of the FastQC stuff is currently located. The "R" workshop includes only the introduction to the command line, but not the wrangling lesson. Intro to the command line does not include any FastQC analysis. @aschuerch could you please confirm there's no FastQC in the intro to shell lesson?

Reply:
@ErinBecker @tracykteal I can confirm that there's no FastQC in intro to shell

Reply:
Thanks, I see now that it's in wrangling-genomics. I was thinking that the FASTQC component from wrangling-genomics would still be in the R version of the workshop. Reading through the updated shell lesson though, I see that doesn't make sense and it should just stay where it is in genomics-wrangling.  

So, maybe that second description could be 2) performing quality filtering and variant calling to learn to automate bioinformatics workflows at the command line. Or something like that. Making it a little more general about the bioinformatics pipelines and automation might be good for people understanding what this version is about.